src/Raven.Client/Documents/Commands/KillOperationCommand.cs

```diff
@@ -8,9 +8,10 @@ public class KillOperationCommand : RavenCommand
     {
         private readonly long _id;
 
-        public KillOperationCommand(long id)
+        public KillOperationCommand(long id, string node = null)
```

Comment:
nodeTag

Reply:
create a separate ctor
this breaks binary compatibility

Reply:
done